src/dispatch-payload.ts

```diff
@@ -142,3 +125,26 @@ function getFunctionCallbackID(
         ?.function_data?.function?.callback_id ?? "";
   }
 }
+
+export function extractBaseHandlerArgsFromPayload(
+  payload: InvocationPayload<ValidInvocationPayloadBody>,
+): BaseHandlerArgs {
+  const { body, context } = payload;
+  const env = context.variables || {};
+  const team_id = context.team_id || "";
+  const enterprise_id = body.enterprise_id ||
+    (body as BaseEventInvocationBody).enterprise?.id || "";
+  const token = (body as FunctionInvocationBody).event?.bot_access_token ||
+    (body as BaseEventInvocationBody).bot_access_token ||
+    context.bot_access_token || "";
+  const inputs = (body as FunctionInvocationBody).event?.inputs ||
+    (body as BaseEventInvocationBody).function_data?.inputs || {};
+  return {
+    body,
```

Comment:
Including `body` in the base handler arguments makes it a new addition for `function_executed` events (previously they did not get a `body` argument). I see that as a good thing, but let me know if you disagree.

Reply:
There is indeed a lot in `body` for function execution events. And i think we intentionally didn't include or type it because it has been in flux - but no longer. There is nothing unsafe there as far as I can tell.

I don't think we should type it, nor document it, if/when that becomes a requirement. But until then, it simplifies the code. I will move forward with this in the mean time unless you feel I should keep it out?

Reply:
I'm good to move forward with it in. Worth acknowledging that this will mean that a developer could access `body` if they realized it's there (might need a ts-ignore), but since there's nothing unsafe that doesn't seem problematic to me

Reply:
Yeah I had to add a `@ts-ignore` to get it show up at runtime.
Discussed offline, is safe, only an additional event ID is exposed on top of what is already exposed elsewhere.